src/Kiota.Builder/Plugins/AdaptiveCardTemplate.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.Extensions.Logging;
+
+namespace Kiota.Builder.Plugins
+{
+
+    internal class AdaptiveCardTemplate
+    {
+        private readonly ILogger<KiotaBuilder> Logger;
+        private string AdaptiveCard => @"
+            {
+                ""$schema"": ""http://adaptivecards.io/schemas/adaptive-card.json"",
+                ""type"": ""AdaptiveCard"",
+                ""version"": ""1.0"",
+                ""body"": [
+                    {
+                        ""type"": ""Container"",
+                        ""items"": [
+                            {
+                                ""type"": ""TextBlock"",
+                                ""text"": ""This is your adaptive card template"",
+                                ""weight"": ""bolder"",
+                                ""size"": ""medium""
+                            },
+                            {
+                                ""type"": ""ColumnSet"",
+                                ""columns"": [
+                                    {
+                                        ""type"": ""Column"",
+                                        ""width"": ""auto"",
+                                        ""items"": [
+                                            {
+                                                ""type"": ""Image"",
+                                                ""url"": ""https://github.com/microsoft/kiota/blob/main/vscode/microsoft-kiota/images/logo.png"",
```

Comment:
We have updated the URL.. it should be https://github.com/microsoft/kiota/blob/main/vscode/microsoft-kiota/images/logo.png?raw=true

the raw=true was missing